frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/index.tsx

```diff
@@ -106,53 +122,63 @@ const Content: FC<Props> = ({ onClose, name, dataAssets }) => {
     handleAnalytics(EVENTS.DOWNLOAD_DATA_FORMAT_CLICKED, format);
   };
 
-  const renderDownload = () => {
-    return (
-      <DownloadButton
-        disabled={!downloadLink || isLoading}
-        data-testid="download-asset-download-button"
-        href={downloadLink}
-        intent={Intent.PRIMARY}
-        onClick={() => handleAnalytics(EVENTS.DOWNLOAD_DATA_COMPLETE)}
-      >
-        Download
-      </DownloadButton>
-    );
-  };
-
   const availableFormats = dataAssets.map((dataAsset) => dataAsset.filetype);
 
   return (
     <>
-      <div className={Classes.DIALOG_BODY}>
-        <Wrapper>
-          <Name name={name} />
-          <DataFormat
-            handleChange={handleChange}
-            isDisabled={isLoading}
-            selectedFormat={selectedFormat}
-            availableFormats={availableFormats}
-          />
-          <Details
-            isLoading={isLoading}
-            fileSize={fileSize}
-            selected={Boolean(fileSize)}
-          />
-          {downloadLink && !isLoading && (
-            <CurlLink
-              fileName={fileName}
-              handleAnalytics={() => handleAnalytics(EVENTS.DOWNLOAD_DATA_COPY)}
-              link={downloadLink}
+      <DialogTitle title="Download Dataset" />
+      <DialogContent>
+        {isError && <div>Dataset download is currently not available.</div>}
+        {isLoading && <Loader sdsStyle="minimal" />}
+        {!isError && !isLoading && (
+          <>
+            <Name name={name} />
+            <DataFormat
+              availableFormats={availableFormats}
+              handleChange={handleChange}
+              isDisabled={isDownloadLinkLoading}
+              selectedFormat={selectedFormat}
+            />
+            <Details
+              curlPreview={
+                downloadLink &&
+                !isDownloadLinkLoading && (
+                  <CurlLink
```

Comment:
`CurlLink` is now passed into `Details` due to a change in the DOM structure driven by updated styles.